### PR TITLE
Add values.yml and update seeding to use it

### PR DESCRIPTION
**Pull request by KComrade53**
## Description

Add a new values.yml file under `data/values` for storing values and their ids instead of directly under attributes. The serializer and seeding logic has been updated to reflect these changes.

The yaml files under `data/` are also updated as part of this PR with the output of our new parser.

## Questions for reviewers

- Do I need to increment the version and add to the changelog as part of this PR?

### CODE REVIEW

**Comment by elsom25** [comment]
Should we also pop attributes and values out of folders?

File: test/unit/property_value_test.rb [hunk 1]
```diff
@@ -12,20 +12,20 @@ def teardown
   end
 
   test "default ordering is alphabetical with 'other' last" do
-    red = value!(id: 11, name: "Red")
-    zoo = value!(id: 12, name: "Zoo")
-    other = value!(id: 10, name: "Other")
+    red = value!(id: 11, name: "Red", friendly_id: "test__red")
```
Comment (tylerrowsell) [inline test/unit/property_value_test.rb:15]: Curious why we chose double underscore?

**Review by tylerrowsell — APPROVED** [review]

**Comment by KComrade53** [comment]
Looks like data is still good

Screenshot 2024-03-25 at 10 20 06 AM

File: db/schema.rb [hunk 2]
```diff
@@ -26,9 +29,9 @@
   end
   create_table :properties_property_values, id: false, force: :cascade do |t|
     t.integer(:property_id, null: false)
-    t.integer(:property_value_id, null: false)
+    t.string(:property_value_friendly_id, null: false)
 
-    t.index([:property_id, :property_value_id], unique: true)
+    t.index([:property_id, :property_value_friendly_id], unique: true)
     t.index(:property_value_id)
```
[the remark this answers is not in the document]
Comment (elsom25) [reply db/schema.rb:35]: Ensures lookups are fast

**Pull request merged**